src/tests.c

```diff
@@ -4918,7 +4918,8 @@ int main(int argc, char **argv) {
         }
     } else {
         FILE *frand = fopen("/dev/urandom", "r");
-        if ((frand == NULL) || fread(&seed16, sizeof(seed16), 1, frand) != sizeof(seed16)) {
+        if ((frand == NULL) || fread(&seed16, 1, sizeof(seed16), frand) != sizeof(seed16)) {
+            fprintf(stderr, "WARNING: could not read 16 bytes from /dev/urandom; falling back to insecure PRNG\n");
```

Comment:
src/tests.c: In function ‘main’:
src/tests.c:4923:13: warning: ISO C90 forbids mixed declarations and code [-Wdeclaration-after-statement]
             uint64_t t = time(NULL) * (uint64_t)1337;

Reply:
@hdon to fix this, move the fprint down a line. Then we should be good to merge.